infrastructure/http/YamlHTTPRequestParser.go

```diff
@@ -13,20 +13,29 @@ type YamlHTTPRequestParser struct{}
 //Parse ...
 func (instance YamlHTTPRequestParser) Parse(input map[string]interface{}) core.Action {
```

Comment:
No tests have changed for these changes, so is that because we have no tests or the tests did not need to change?